File: Compiladores(ano passado)/Projecto Ano Passado/submissao_final/at/NodeUminus.cpp
#include "NodeUminus.hpp"

void NodeUminus::accept(Compiler& c) { c.nodeUminus(*this); }
void NodeUminus::print(std::ostream *out) {
	if (out) (*out) << "NodeUminus: ";
	NodeUnary::print(out);
}

File: Compiladores(ano passado)/compact12b/compact/java/NodeUminus.java
import java.util.*;

public class NodeUminus extends NodeUnary {
	NodeUminus(Node arg1) { super(arg1); }
	NodeUminus(Node arg1, int lineno) { super(arg1, lineno); }
	public void accept(Compiler c) { c.nodeUminus(this); }
}
